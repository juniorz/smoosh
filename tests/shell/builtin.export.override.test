unset x
$TEST_UTIL/getenv x
x=4
$TEST_UTIL/getenv x
export x=5
$TEST_UTIL/getenv x
x=6 $TEST_UTIL/getenv x
echo x is ${x-unset}
