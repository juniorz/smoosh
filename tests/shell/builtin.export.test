cat >scr <<'EOF'
echo ${var-unset}
EOF
$TEST_SHELL scr
var=hi
$TEST_SHELL scr
var=here $TEST_SHELL scr
export var=bye
$TEST_SHELL scr
