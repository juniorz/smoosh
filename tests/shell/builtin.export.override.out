x is unset
x is unset
x='5'
x='6'
x is 5
